models/Document/Editable/Video.php

```diff
@@ -1052,6 +1052,14 @@ private function updateAllowedTypesFromConfig(array $config): void
         }
     }
 
+    /**
+     * @return string
+     */
```

Comment:
```suggestion

```

Reply:
This phpDoc can be removed since we have already typed the return type here https://github.com/pimcore/pimcore/pull/12982/files#diff-9cba4e2ecaf34598897940fdff3416069ea2708f77e953bf87f08b2529651007R1055-R1057:~:text=function%20getUniqId()%3A-,string,-%7B

Reply:
I was just following the code style such as 

https://github.com/pimcore/pimcore/blob/d49f5a5bc9c62a4c75808d877187ee67d36190c1/models/Document/Editable/Video.php#L1022-L1025

Reply:
@NiklasBr yes, we decided to get rid off duplicate info recently, so we can already fix the changed files and later on rest of things will be taken care of. thanks!